lib/internal/event_target.js

```diff
@@ -591,49 +591,59 @@ class EventTarget {
    *   signal?: AbortSignal
    * }} [options]
    */
-  addEventListener(type, listener, options = kEmptyObject) {
+  addEventListener(type, listener, options) {
```

Comment:
This is probably needed to keep `EventTarget.prototype.addEventListener.length === 2`

Reply:
You can default it to `undefined` if that's important for the perf.